mne/epochs.py

```diff
@@ -1233,6 +1242,15 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
+                if any([k_e not in epochs.event_id for k_e in key]):
+                    key = [k for k in epochs.event_id.keys()
+                           if all([set(k_i.split('/')).issubset(k.split("/"))
+                                   for k_i in key])]
+                    if len(key) == 0:
+                        raise KeyError('Attempting selection of events via '
+                                       'multiple/partial matching, but no '
+                                       'event matches all criteria.')
```

Comment:
...and in cases where you *really* wanted to select one versus the other you could still ask explicitly for `a/b` or `b/a`, and it would find just that one.